### PR TITLE
add a new module to create isis instance.

### PR DESCRIPTION
**Pull request by yuandongx**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

lib/ansible/modules/network/cloudengine/ce_is_is_instance.py
##### ADDITIONAL INFORMATION

```paste below

```
### test case
```
  tasks:
  - name: Set isis process 1
    ce_is_is_instance:
      instance_id: 1
      vpn_name: a
      state: present

```
### log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "instanceId": "1",
            "vpnName": "a"
        }
    },
    "existing": {
        "instance": {}
    },
    "invocation": {
        "module_args": {
            "instance_id": 1,
            "state": "present",
            "vpn_name": "a"
        }
    },
    "proposed": {
        "instance_id": 1,
        "state": "present",
        "vpn_name": "a"
    },
    "updates": [
        "isis 1",
        "vpn-instance a"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```
### display on device
```

[~8850_130.12-isis-1]di th
#
isis 1 vpn-instance a
#
return
[~8850_130.12-isis-1]

```
```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+      state: present
+
+  - name: check vpn name
+    ce_is_is_instance:
+      instance_id: 22
+      vpn_name: vpn1234567896321452212221556asdasdasdasdsadvdv
+      state: present
+'''
+
+RETURN = '''
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: r'''

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+    - Manages  isis process id, creates a isis instance id or deletes a process id on HUAWEI CloudEngine devices.
+options:
+  instance_id:
+    description:
+      - Specifies the id of a isis process.The value is a number of 1 to 4294967295.
+    required: true
+    type: int
+  vpn_name:
+    description:
+      - VPN Instance, Associate the VPN instance with the corresponding IS-IS process.
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: Associate -> associate

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    required: false
+    type: str
+  state:
+    description:
+      - Determines whether the config should be present or not on the device.
+    required: false
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: required: false is default behavior.

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+options:
+  instance_id:
+    description:
+      - Specifies the id of a isis process.The value is a number of 1 to 4294967295.
+    required: true
+    type: int
+  vpn_name:
+    description:
+      - VPN Instance, Associate the VPN instance with the corresponding IS-IS process.
+    required: false
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: required: false is default behavior.

**Review by yuandongxx — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py (hunk 4 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: Done

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py (hunk 3 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: Done

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py (hunk 2 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: Done

File: lib/ansible/modules/network/cloudengine/ce_is_is_instance.py (hunk 1 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_is_is_instance.py]: Done

**Comment by xuxiaowei0512** [comment]
ready_for_review

**Comment by hwDCN** [comment]
shipit

**Review by yuandongxx — APPROVED** [review]

**Comment by hwDCN** [comment]
shipit

**Pull request merged**